README.md

```diff
@@ -69,7 +69,7 @@ As a Jira administrator, you have two steps to configure the plugin:
   - Enabling development mode allows you to install apps that are not from the Atlassian Marketplace.
   - Mattermost has opted not to publish to Atlassian Marketplace, as we don’t have an efficient way to provide a callback URL for the app in the marketplace, to enable user-specific interactions between the Mattermost server and Jira cloud instance.
 
-3. Click **Upload app**, then enter the Atlassian Connect app descriptor in the form https://<SITEURL>/plugins/jira/atlassian-connect.json where `SITEURL` is your [Mattermost Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url). Select **Upload**.
+3. Click **Upload app**, then enter the Atlassian Connect app descriptor in the form https://SITEURL/plugins/jira//ac/atlassian-connect.json where `SITEURL` is your [Mattermost Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url). Select **Upload**.
```

Comment:
remove the `//ac/` -> `/ac/`. I thought I fixed the display in the message, will check.

Reply:
When I set it up this morning, it worked with the double backslash. Didn't try single backslash. Just FYI @levb

Reply:
I fixed it in https://github.com/mattermost/mattermost-plugin-jira/pull/47. Going to merge as is since it's inconsequential and the UI/doc will be changed for the release anyway.